allennlp/training/metrics/quoref_em_and_f1.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Union
+
+from overrides import overrides
+
+from allennlp.training.metrics.metric import Metric
+from allennlp.training.metrics.drop_em_and_f1 import DropEmAndF1
+
+
+@Metric.register("quoref")
+class QuorefEmAndF1(DropEmAndF1):
```

Comment:
Probably better to just instantiate the `DropEmAndF1` inside of `__init__` and call it in `__call__`, instead of inheriting.  The end result is the same, and you have more flexibility later.

Reply:
Even better, can you just have the model format the ground truth in the way that the other metric expects, and remove the need for this class entirely?

Reply:
Agreed. Removing this class.